plugins/azure/README.md

```diff
@@ -0,0 +1,70 @@
+# Azure Plugin
+
+![preview of Azure table](docs/functions-table.png)
+
+_Inspired by [roadie.io AWS Lamda plugin](https://roadie.io/backstage/plugins/aws-lambda/)_
+
+## Features
+
+- Azure overview table
+
+## Plugin Setup
+
+The following sections will help you get the Azure Functions plugin setup and running
+
+### Azure Functions Backend
+
+You need to set up the Azure backend plugin before you move forward with any of these steps if you haven't already.
+
+### Entity Annotation
+
+To be able to use the Azure plugin you need to add the following annotation to any entities you want to use it with:
+
+```yaml
+portal.azure.com/resource-name: <resource-name>
+```
+
+`<resource-name>` can be an exact or partial name for the site.
+
+Example of Partial Matching:
+
+Let's say you have a number of functions apps, spread out over different regions (and possibly different subscriptions), and they follow a naming convention:
+
+```
+func-testapp-eu
+func-testapp-ca
+func-testapp-us
+```
+
+The annotation you will use to have the three functions' app appear in the overview table would look like this:
+
+```yaml
+portal.azure.com/resource-name: func-testapp
```

Comment:
Thinking about this some more - a generic annotation name could start to cause conflicts:
* What type of resource is this - a function app, keyvault etc? 
* Can an entity have multiple different types of azure resources associated with it.  (e.g. a function app and a keyvault).

I'm sort of leaning towards having the annotation be specific to a resource type 
* `azure.com/function-names`
* `azure.com/microsoft.web/sites` (using arm types)

Reply:
You're right, I decided to go with the arm types for annotation name. thanks!